secgroup_ingress-from-proxy.tf

```diff
@@ -0,0 +1,27 @@
+resource "openstack_networking_secgroup_v2" "ingress-from-proxy" {
+  name                 = "ingress-from-proxy"
+  description          = "[tf] Ingress from proxy"
+  delete_default_rules = "true"
+}
+
+resource "openstack_networking_secgroup_rule_v2" "ingress-from-proxy_ingress" {
+  direction         = "ingress"
+  ethertype         = "IPv4"
+  security_group_id = openstack_networking_secgroup_v2.ingress-from-proxy.id
+  protocol          = "tcp"
+  port_range_min    = 5555
+  port_range_max    = 5555
+  remote_ip_prefix  = "132.230.103.37/32"
```

Comment:
I think this IP also doesn't belong to us anymore.
Maybe we could delete the whole file